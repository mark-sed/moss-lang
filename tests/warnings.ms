"start\n"
corr = -9223372036854775807
corr = 9223372036854775807
incor = -9223372036854775808
incor2 = 9223372036854775808
incor
"\n"
incor2
"\n"

fincor = 1e500
fincor
"\n"
fincor2 = -2.5e-500
fincor2
"\n"
-1e500
"\n"