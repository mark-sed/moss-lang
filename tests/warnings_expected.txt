warning: tests/warnings.ms:4:29:
      | Value '9223372036854775808' cannot fit into Int. [WDx1]
      | 
    4 | incor = -9223372036854775808 
      |                                    ^
warning: tests/warnings.ms:5:29:
      | Value '9223372036854775808' cannot fit into Int. [WDx1]
      | 
    5 | incor2 = 9223372036854775808 
      |                                    ^
warning: tests/warnings.ms:11:15:
      | Value '1e500' is too big or too close to 0 to fit into Float. [WDx2]
      | 
   11 | fincor = 1e500 
      |                      ^
warning: tests/warnings.ms:14:20:
      | Value '2.5e-500' is too big or too close to 0 to fit into Float. [WDx2]
      | 
   14 | fincor2 = -2.5e-500 
      |                           ^
warning: tests/warnings.ms:17:7:
      | Value '1e500' is too big or too close to 0 to fit into Float. [WDx2]
      | 
   17 | -1e500 
      |              ^
start
-9223372036854775807
9223372036854775807
inf
-0.000000
-inf
